hcmanager/utils.go

```diff
@@ -100,61 +86,51 @@ func SetStatus(hwc *hwcc.HardwareClassification,
 
 //ValidateExtractedHardwareProfile it will validate the extracted hardware profile and log the warnings
 func (mgr HardwareClassificationManager) ValidateExtractedHardwareProfile(extractedProfile hwcc.HardwareCharacteristics) error {
```

Comment:
You could also in all those checks verify that the mins are always smaller than the max, so that you would not have a situation where minCPU=4 and MaxCPU=3 for example, but also for ram, nics etc.

In addition, this kind of validation should probably be done in a webhook rather, when the resource is being created.

Reply:
I will add validations using webhook. That will be better approach.
@ravipwaghmare  We should address in separate PR instead adding it here as there is other validations also to be addressed as part of that PR.